src/components/Loans/ClosedLoans/ClosedLoans.js

```diff
@@ -73,6 +73,23 @@ class ClosedLoans extends React.Component {
     this.handleOptionsChange = this.handleOptionsChange.bind(this);
     this.getFeeFine = this.getFeeFine.bind(this);
     this.anonymizeLoans = this.anonymizeLoans.bind(this);
+    this.buildRecords = this.buildRecords.bind(this);
+
+    this.headers = ['action', 'dueDate', 'loanDate', 'returnDate', 'systemReturnDate', 'item.barcode', 'item.callNumber', 'item.contributors',
+      'item.holdingsRecordId', 'item.instanceId', 'item.status.name', 'item.title', 'item.materialType.name',
+      'item.location.name', 'metaData.createdByUserId', 'metadata.updatedDate', 'metadata.updatedByUserId', 'loanPolicyId'];
+
+    this.headers = ['action', 'dueDate', 'loanDate', 'returnDate', 'systemReturnDate', 'item.barcode', 'item.callNumber', 'item.contributors',
+      'item.holdingsRecordId', 'item.instanceId', 'item.status.name', 'item.title', 'item.materialType.name',
+      'item.location.name', 'metaData.createdByUserId', 'metadata.updatedDate', 'metadata.updatedByUserId', 'loanPolicyId'];
+
+    // Map to pass into exportCsv
+    this.columnHeadersMap = this.headers.map(item => {
+      return {
+        label: stripes.intl.formatMessage({ id: `ui-users.${item}` }),
+        value: item
+      };
+    });
 
     this.sortMap = {
       [stripes.intl.formatMessage({ id: 'ui-users.loans.columns.title' })]: loan => _.get(loan, ['item', 'title']),
```

Comment:
No need for the key to be internationalized. 
We need to add this column internationalization to columnMapping

Reply:
@Godlevskyi  We need the key to be internationalized here as we need the value from the translations file to be the label for our csv columns here. Not sure if you are referring to MCL when you say columnMapping here? as we use json2csv module for the csv export and it would need a label, value pair as can be seen  in this example: https://www.npmjs.com/package/json2csv#example-3

Reply:
I thing it we do not need to internathionalize the key values as we have all the translations for the MCL in columnMapping. you can move it from the render up and use it for the header lables
```
    const columnMapping = {
       'title': this.props.stripes.intl.formatMessage({ id: 'ui-users.loans.columns.title' }),
       'barcode': this.props.stripes.intl.formatMessage({ id: 'ui-users.loans.columns.barcode' }),
       'Fee/Fine': this.props.stripes.intl.formatMessage({ id: 'ui-users.loans.columns.feefine' }),
       'loanDate': this.props.stripes.intl.formatMessage({ id: 'ui-users.loans.columns.loanDate' }),
       'dueDate': this.props.stripes.intl.formatMessage({ id: 'ui-users.loans.columns.dueDate' }),
       'returnDate': this.props.stripes.intl.formatMessage({ id: 'ui-users.loans.columns.returnDate' }),
       'renewals': this.props.stripes.intl.formatMessage({ id: 'ui-users.loans.columns.renewals' }),
       'Call Number': this.props.stripes.intl.formatMessage({ id: 'ui-users.loans.details.callNumber' }),
       'Contributors': this.props.stripes.intl.formatMessage({ id: 'ui-users.loans.columns.contributors' }),
     };
```
Can you use it for the csv headers also, like: columnMapping['title']?

Reply:
@Godlevskyi Then thing here is the headers that export uses don't match with the ones in columnMapping. We have more headers in the export than in the columnMapping.(FYI: we have 16 columns in export vs 9 in columnMapping). But, yes we can definitely move the columnMapping into the constructor and add in all the additional translations there and use something like columnMapping['title']. But, I am not sure if that would make the code look cleaner? That would mean almost 20 to 25 lines of translations for the columnMapping variable.
Is that what you are suggesting here?

Reply:
@aditya-matukumalli, thanks for the clarification regarding the export.
But my point was more about the MCL implementation. 
I do not think that it is correct to have translations in the sortMap and sortOrder, as we have mappings in the columnMapping.

Sorry, this comment was not related to the current PR but more to the component itself.
Thanks

Reply:
@Godlevskyi I think its better if I file a new JIRA to remove the translations from sortMap and sortOrder and possibly other places and work on it. Thanks

Reply:
@aditya-matukumalli Great! thank you!